dvc/remote/gdrive/utils.py

```diff
@@ -0,0 +1,55 @@
+import functools
+import os
+import threading
+
+from dvc.progress import progress
+
+
+class track_progress(object):
```

Comment:
I believe it's not PEP8

Reply:
Its constructor is used like a callable, and this is acceptable by PEP8. But it would be good to add a docstring at least (though as of my sense this naming scheme actually works as a replacement for such a docstring).

https://www.python.org/dev/peps/pep-0008/#class-names:

> The naming convention for functions may be used instead in cases where the interface is documented and used primarily as a callable.

But this is actually a good question, is it a good practice to "hide" the fact of creating the proxy object here, or would it be better to show it explicitly as it will be with the regular CapWords class naming.